sympy/utilities/tests/test_wester.py

```diff
@@ -2909,15 +2909,13 @@ def test_Y1():
     F, _, _ = laplace_transform(cos((w - 1)*t), t, s)
     assert F == s/(s**2 + (w - 1)**2)
 
-
 def test_Y2():
     t = symbols('t', positive=True)
     w = symbols('w', real=True)
     s = symbols('s')
     f = inverse_laplace_transform(s/(s**2 + (w - 1)**2), s, t)
     assert f == cos(t*(w - 1))
 
-
 def test_Y3():
```

Comment:
```suggestion

def test_Y3():
```
Please revert back these changes

Reply:
Thanks; the plan is that when the chaotic phase of the development is finished, I will revert all files except transforms.py and test_transforms.py, and then squash commits.